cli/cli_test.go

```diff
@@ -118,5 +122,23 @@ var _ = Describe("CLI", func() {
 			waitForLocal.Done()
 			waitForThread.Wait()
 		})
+
+		It("Should handle sending post-migration event to forced schemas", func() {
+			set := flag.NewFlagSet("", flag.PanicOnError)
+			cli.StringFlag{Name: flagForcedSchemas, Value: "force_schema"}.Apply(set)
+			cli.BoolTFlag{Name: flagEmitPostMigrationEvent}.Apply(set)
+			cli.StringFlag{Name: flagConfigFile, Value: "../tests/test_forced_post_migrate_config.yaml"}.Apply(set)
+			cli.DurationFlag{Name: flagPostMigrationEventTimeout, Value: time.Duration(60) * time.Second}.Apply(set)
+			context := cli.NewContext(nil, set, &cli.Context{})
+			configFile := context.String(flagConfigFile)
+			loadConfig(configFile)
+			Expect(migration.LoadConfig(configFile)).To(Succeed())
+			actionPostMigrateEvent(context)
+			sync, err := sync_util.CreateFromConfig(util.GetConfig())
```

Comment:
just by looking at this test it's hard to tell which part is setup, which is exercise and which is verify. please restructure this test. either extract helper methods or add some whitespace/comments.

Reply:
ok, fixed

Reply:
ok, fixed